ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -702,3 +725,51 @@ def unpatch():
     core.dispatch("langchain.unpatch", tuple())
 
     delattr(langchain, "_datadog_integration")
+
+
+@with_traced_module
```

Comment:
Best practice is to keep the `patch/unpatch` methods below the `traced_` methods